hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/auth/delegation/ITestSessionDelegationInFileystem.java

```diff
@@ -330,11 +329,10 @@ public void testDelegatedFileSystem() throws Throwable {
         + " if role restricted, permissions are tightened.");
     S3AFileSystem fs = getFileSystem();
     // force a probe of the remote FS to make sure its endpoint is valid
-    // TODO: Previously a call to getObjectMetadata for a base path, ie with an empty key would
-    //  return some metadata. (bucket region, content type). headObject() fails without a key, check
-    // how this can be fixed.
-    // fs.getObjectMetadata(new Path("/"));
-    readLandsatMetadata(fs);
+    // TODO: Check what should happen here. Calling headObject() on the root path fails in V2,
```

Comment:
should we call headBucket here? or just remove?

Reply:
it's a question I need to ask Steve